neocore/UIntBase.py

```diff
@@ -8,21 +8,32 @@ class UIntBase(SerializableMixin):
     __hash = None
 
     def __init__(self, num_bytes, data=None):
+        """
+        Create an instance.
+
+        Args:
+            num_bytes: (int) the length of data in bytes
+            data: (bytes, bytearray; optional) the data to be converted to UIntBase
```

Comment:
```suggestion
            data: (bytes, bytearray; optional) the raw data
```
My reason for changing this is that "converted" suggests manipulation of the data. For this class the actual underlying data is not manipulated. We actually put a wrapper around it to expose convenience functions for parsing data, recognising data of specific lengths as specific types (e.g. 20 bytes = UInt160, 32 bytes is UInt256), and printing them in a user friendly format. But the data that is worked on internally is not modified.